gulp/tasks/watch.js

```diff
@@ -3,10 +3,18 @@
 
 const gulp = require( 'gulp' );
 const config = require( '../config' );
```

Comment:
Let's nix the config here too. I mean look at `config.legacy.cwd` all it resolves to is `paths.legacy`!